tests/test_validations.py

```diff
@@ -264,6 +264,30 @@ def test_dates_before_current():
   errors = validation.validate_dates_before_current(gedcom)
   assert errors[0] == 'Error: US01: The Individual I01\'s deathdate 2025-10-10 is after the current date'
   assert len(errors) == 1
+
+def test_correct_gender():
+  # Husband is female or anything other than male
+  families = [ Family('F01', married=datetime(1999, 9, 9), husband_id='I01', wife_id='I02')]
+  individuals = [
+    Individual('I01', spouses=['F01'], sex='F'),
+    Individual('I02', spouses=['F01'], sex='F')
+  ]
+  gedcom = Gedcom(individuals = individuals, families = families)
+  errors = validation.validate_correct_gender(gedcom)
+  assert errors[0] == 'Error: US21: Husband I01 in Family F01 should be male'
+  assert len(errors) == 1
+
+  # Wife is male or anything other than female
+  families = [ Family('F01', married=datetime(1999, 9, 9), husband_id='I01', wife_id='I02')]
+  individuals = [
+    Individual('I01', spouses=['F01'], sex='F'),
+    Individual('I02', spouses=['F01'], sex='M')
```

Comment:
You have two errors here. The husband is female and wife is male. You either change this to be one error or assert both errors.